Code Your Own Adventure Test

```diff
@@ -0,0 +1,79 @@
+prompt confirm("I am ready to play!")
+ii. Old enough to play? 2/7
+
+confirm("I am ready to play!")
+var age = prompt("What's your age");
+if (age < 13)
+{
+    console.log("Youallowed to play but we don't take any responsibility.")
+}
+else
+{
+    console.log("Play)
+}
+
+
+confirm("I am ready to play!")
+var age = prompt("What's your age");
+if (age < 13)
+{
+    console.log("Youallowed to play but we don't take any responsibility.")
+}
+else
+{
+    console.log("Play)
+}
+
+console.log("Youat a Justin Bieber concert, and you hear this lyric 'Lace my shoes off, start racing.'")
+iv. First move! 4/7
+
+confirm("I am ready to play!");
+var age = prompt("What's your age");
+if (age < 13)
+{
+    console.log("Youallowed to play but we don't take any responsibility.");
+}
+else
+{
+    console.log("Play);
+}
+
+console.log("Youat a Justin Bieber concert, and you hear this lyric 'Lace my shoes off, start racing.'");
+console.log("Suddenlyber stops and says, 'Who wants to race me?'");
+var userAnswer = prompt("Do you want to race Bieber on stage?");
+v. The story heats up! 5/7
+
+confirm("I am ready to play!");
+var age = prompt("What's your age");
+if (age < 13)
+{
+    console.log("Youallowed to play but we don't take any responsibility.");
+}
+else
+{
+    console.log("Play);
+}
+
+console.log("Youat a Justin Bieber concert, and you hear this lyric 'Lace my shoes off, start racing.'");
+console.log("Suddenlyber stops and says, 'Who wants to race me?'");
+var userAnswer = prompt("Do you want to race Bieber on stage?");
+if (userAnswer == "yes")
+{
+    console.log("YouBieber start racing. It's neck and neck! You win by a shoelace!");
+}
+else
+{
+    console.log("OhBieber shakes his head and sings 'I set a pace, so I can race without pacing.'");
+}
+vi. Asking for feedback 6/7
+
+confirm("I am ready to play!");
+var age = prompt("What's your age");
+if (age  8)
+{
+    console.log("Thank We should race at the next concert!");
+}
+else
+{
+    console.log("I'llcticing coding and racing.");
+}
```

Comment:
looks like you're off to a good start! Name this file `.js` to show syntax highlighting